### DISCUSSION TITLE
[Feature] Drag and drop folder to upload all images with in

### DISCUSSION THREAD

**Discussion by Aseeef** [immich-app/immich#11209] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In google photos, you can drag and drop an entire folder which may have nested directories with lots of photos and GPhotos uploads them all. Asking for a similar feature here.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5938
